contracts/merkle-distributor/MerkleDistributor.sol

```diff
@@ -0,0 +1,257 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
When you say "based on" do you mean directly copied? Or are there changes? If there are changes, we will likely need to have this re-audited.

Reply:
The contract itself is identical